packages/studio-web/src/app/upload/upload.component.ts

```diff
@@ -421,7 +421,7 @@ Please check it to make sure all words are spelled out completely, e.g. write "4
               }
               return throwError(() => err);
             } else {
-              return of(err);
+              return throwError(() => err);
```

Comment:
Hm. but doesn't this cause retry to trigger? I'll test this.

Reply:
yea, on my machine this triggers the error message three times since any error thrown will cause `retry` to trigger:

![image](https://github.com/ReadAlongs/Web-Component/assets/14815080/f3045276-e24d-4059-8405-1dc9ce681db0)